samples/sample_encode/include/pipeline_encode.h

```diff
@@ -291,11 +291,11 @@ class CEncodingPipeline
 
     void SetNumView(mfxU32 numViews) { m_nNumView = numViews; }
     virtual void  PrintInfo();
-
+#if defined (ENABLE_V4L2_SUPPORT)
```

Comment:
Silly question: do we need this #if...#endif

Reply:
I think so, also it`s way to fix build w/ and w/o V4L2